### PR TITLE
fix(vcs): no files were found when dotIgnoreFiles was set to empty list

### PR DESCRIPTION
**Pull request by edvald**
**What this PR does / why we need it**:

If you set `dotIgnoreFiles: []` in your project config, no modules would be found.

**Which issue(s) this PR fixes**:

Just came across this myself.

### CODE REVIEW

File: garden-service/src/vcs/git.ts [hunk 1]
```diff
@@ -166,16 +166,20 @@ export class GitHandler extends VcsHandler {
       // We push to the output array when all ls-files commands "agree" that it should be included,
       // and it passes through the include/exclude filters.
       if (
-        paths[resolvedPath] === this.ignoreFiles.length
+        paths[resolvedPath] >= this.ignoreFiles.length
```
Comment (eysi09) [inline garden-service/src/vcs/git.ts]: Why this change?

Comment (edvald) [reply garden-service/src/vcs/git.ts]: Otherwise the fix doesn't work when there are no ignore files, since the count (1) is higher than the number of ignore files (0).
